interactions/api/http.py

```diff
@@ -609,7 +609,7 @@ async def get_guild_widget_settings(self, guild_id: int) -> dict:
 
     async def get_guild_widget_image(self, guild_id: int, style: Optional[str] = None) -> str:
         """
-        Get a url representing a png image widget for the guild.
+        Get an url representing a png image widget for the guild.
```

Comment:
`a url` is correct

Reply:
Could you explain me why?

Reply:
It depends on the sound, not vowel, for whether you use an
URL is pronounced yoo-arr-ell, and the y sound makes it a url, not an url

https://www.latimes.com/socal/glendale-news-press/opinion/tn-gnp-a-word-please-theres-a-sound-reason-to-use-a-or-an-20150313-story.html 
here's an article I found that explains it

Reply:
I knew it depends on the sound but I didn't know that it's pronounced as "yoo"

Thanks!